src/ls/wsjsapi.js

```diff
@@ -479,16 +479,13 @@ var Exasol = function(url, user, pass, onconnect, onerror) {
         context.connection.send(reqdata);
     };
 
-    context.fetch = function(res, startPosition, numBytes, onResponse, onerror) {
+    context.fetch = function(resultSetHandle, startPosition, numBytes, onresponse, onerror) {
```

Comment:
just curious why the `onresponse` rename?

Reply:
yep. All other callbacks are named in lowercase entirely, so that `R` really stuck out